Add check for null values for table visualisation

marthasharkey opened the pull request:
### Pull Request Description

adds a check for any null values 

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [ ] The documentation has been updated, if necessary.
- [ ] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [ ] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [ ] Unit tests have been written where possible.

marthasharkey committed:
add check for null values

marthasharkey committed:
add check for null values

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/enso-org/enso?pullRequest=10114) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/enso-org/enso?pullRequest=10114) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: marthasharkey<br/>:x: Martha King<br/><hr/>**Martha King** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/enso-org/enso?pullRequest=10114) it.</sub>

marthasharkey committed:
remove log

marthasharkey committed:
remove log

Frizi commented on app/gui2/src/components/visualizations/TableVisualization.vue:
```diff
@@ -380,8 +380,10 @@ watchEffect(() => {
       const needsGrouping = rowData.some((row: any) => {
-        const value = typeof row[header] === 'object' ? row[header].value : row[header]
-        return value > 9999
+        if (row[header]) {
```
```suggestion
        if (header in row && row[header] != null) {
```

Frizi reviewed (changes requested):
The current check is not a strict null check. Since we work with arbitrary data, we need to handle cases like 0 or empty string correctly.

Commit:
Update app/gui2/src/components/visualizations/TableVisualization.vue

Co-authored-by: Paweł Grabarz <frizi09@gmail.com>

Commit:
Update app/gui2/src/components/visualizations/TableVisualization.vue

Co-authored-by: Paweł Grabarz <frizi09@gmail.com>

radeusgd committed:
Small fixes to Postgres integration (#10105)

- Better message when saving datalink in disabled Output context:
![image](https://github.com/enso-org/enso/assets/1436948/540d615b-79ff-4811-8262-a0475a7b6923)
Before it was:
![image](https://github.com/enso-org/enso/assets/1436948/51198bf1-1e50-41bc-a56b-f829bc32d09a)

- Hack to get Postgres widget to display connection options:
![image](https://github.com/enso-org/enso/assets/1436948/39f3db39-1163-4815-b59f-c629d812e2ab)
Before the `Postgres` constructor was created without any parameters and it was not showing any parameters for modification.

marthasharkey committed:
Merge branch 'wip/mk/visualisation-with-null-values' of https://github.com/enso-org/enso into wip/mk/visualisation-with-null-values
